synapse/models/person.py

```diff
@@ -46,6 +47,60 @@ def seedPersonGuidName(self, prop, valu, **props):
             node = self.core.formTufoByProp('ps:person', None, guidname=valu, **props)
         return node
 
+    @modelrev('ps', 201802281621)
+    def _revModl201802281621(self):
+        '''
+        Combine ps:has* into ps:has
```

Comment:
@invisig0th - do you want this to be `ps:person:has` instead of `ps:has`?

Reply:
lets make sure we use the updated namespace convention